app/controllers/users_controller.rb

```diff
@@ -7,6 +8,18 @@ def show
     issue_listing(@user.issues)
   end
 
+  def feed
+    current_user.check_notifications unless request.xhr?
+    @events = current_user.events.ordered.page(params[:page]).per(3)
+    respond_for_feed(@events)
```

Comment:
There are a lot of bugs with pagination in previous editions of the system. I`ll debug it and if i won`t find it, i do reading in this action